backend/server/users/models.py

```diff
@@ -21,10 +21,15 @@ class User(AbstractUser):
     description = models.TextField(null=True, blank=True)
     user_type = models.CharField(max_length=20, choices=UserType.choices, default=UserType.PERSONNEL)
     insurance_coverage = models.CharField(max_length=20, choices=InsuranceCoverage.choices, blank=True, null=True)
-    photo = models.ImageField(upload_to='profile_photos/', blank=True, null=True)
+    photo = models.ForeignKey(File, blank=True, null=True, on_delete=models.SET_NULL)
+    photo_url = models.URLField(blank=True)
     specialty = models.CharField(max_length=255, blank=True, null=True)
     location = models.CharField(max_length=255, blank=True, null=True)
 
+    @property
+    def photoURL(self) -> str:

```

Comment:
You can change this to 
```suggestion
    def photo_url(self) -> str:

```